### PR TITLE
Rename Facets feature to Filters.

### PR DESCRIPTION
**Pull request by JakePT**
### Description of the Change
Renames the "Facets" feature to "Filters".

The change is made in user-facing copy only. In code the feature is still referred to as Facets, and links to external documentation remain pointed to documentation on Facets. References in the changelog have _not_ been retrospectively updated.

### How to test the Change
On the _ElasticPress > Features_ page, any references to facets should now reference filters.

### Changelog Entry
Changed - Renamed the Facets feature to Filters.

### Credits

Props @JakePT 

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [x] All new and existing tests pass.

### CODE REVIEW

File: includes/classes/Feature/Facets/Facets.php [hunk 1]
```diff
@@ -490,7 +490,7 @@ public function output_feature_box_long() {
 		<p>
 			<?php
 			// translators: URL
-			echo wp_kses_post( sprintf( __( "Adds a <a href='%s'>Facet widget</a> that administrators can add to the website's sidebars (widgetized areas), so that visitors can filter applicable content and search results by one or more taxonomy terms.", 'elasticpress' ), esc_url( admin_url( 'widgets.php' ) ) ) );
+			echo wp_kses_post( sprintf( __( "Adds <a href='%s'>Filter widgets</a> that administrators can add to the website's sidebars (widgetized areas), so that visitors can filter applicable content and search results by one or more taxonomy terms.", 'elasticpress' ), esc_url( admin_url( 'widgets.php' ) ) ) );
```
Comment (JakePT) [inline includes/classes/Feature/Facets/Facets.php]: Does this also need to be updated to mention the Blocks? The widgets link will not work on FSE themes.

File: includes/classes/Feature/WooCommerce/WooCommerce.php [hunk 2]
```diff
@@ -56,7 +56,7 @@ public function __construct() {
 
 		$this->title = esc_html__( 'WooCommerce', 'elasticpress' );
 
-		$this->summary = __( '“I want a cotton, woman’s t-shirt, for under $15 that’s in stock.” Faceted product browsing strains servers and increases load times. Your buyers can find the perfect product quickly, and buy it quickly.', 'elasticpress' );
+		$this->summary = __( '“I want a cotton, woman’s t-shirt, for under $15 that’s in stock.” Filtered product browsing strains servers and increases load times. Your buyers can find the perfect product quickly, and buy it quickly.', 'elasticpress' );
```
Comment (JakePT) [inline includes/classes/Feature/WooCommerce/WooCommerce.php:59]: Unrelated, but here it sounds like our feature strains the server and increased load times?

File: includes/classes/Feature/Facets/Facets.php [hunk 3]
```diff
@@ -486,13 +486,27 @@ public function hide_legacy_widget( $widgets ) {
...
+			$message = sprintf(
+				/* translators: Site Editor URL */
+				__( "Adds <a href='%s'>filter blocks</a> that administrators can add to the website's templates and template parts, so that visitors can filter applicable content and search results by one or more taxonomy terms.", 'elasticpress' ),
+				esc_url( admin_url( 'site-editor.php' ) )
+			);
+		}
+
+		if ( ! isset( $message ) ) {
+			return;
+		}
```
Comment (felipeelia) [inline includes/classes/Feature/Facets/Facets.php:507]: @JakePT this is my suggestion for the problem you brought up before

File: includes/classes/Feature/WooCommerce/WooCommerce.php (hunk 2 above)
Comment (felipeelia) [reply includes/classes/Feature/WooCommerce/WooCommerce.php:59]: @brandwaffle will share with us an updated copy and that will be changed in an upcoming PR.

Comment (brandwaffle) [reply includes/classes/Feature/WooCommerce/WooCommerce.php:59]: "With ElasticPress, filtering WooCommerce product results is fast and easy. Your customers can find and buy exactly what they're looking for, even if you have a large or complex product catalog."

**Pull request merged**